### PR TITLE
bug: fix db test to do month addition properly

### LINKED ISSUE #261: Fix db test bug
It's december, so the poor month math in a db check now fails. It should be handled properly rather than just adding 1 to the current month.

### PR DESCRIPTION
**Pull request by bbangert**
Closes #261.

@jrconlin r?

### CODE REVIEW

File: autopush/tests/test_db.py [hunk 1]
```diff
@@ -56,7 +56,9 @@ def test_get_month(self):
         from autopush.db import get_month
         month0 = get_month(0)
         month1 = get_month(1)
-        eq_(month0.month+1, month1.month)
+        this_month = month0.month
+        next_month = 1 if this_month == 12 else this_month + 1
```
Comment (kitcambridge) [inline autopush/tests/test_db.py]: How about `(this_month % 12) + 1`?

**Comment by kitcambridge** [comment]
r+

File: autopush/tests/test_db.py (hunk 1 above)
Comment (bbangert) [inline autopush/tests/test_db.py]: Mainly because modulus operations aren't as obvious when skimming.

Comment (kitcambridge) [inline autopush/tests/test_db.py]: :+1:

**Pull request merged**